frontend/src/models/models.html

```diff
@@ -84,6 +84,16 @@
       <div v-if="status === 'loading'" class="loader">
         <img src="images/loader.gif">
       </div>
+      <div v-if="outputType =='json' && status != 'loading'">
+        <div v-for="document in documents" @click="$router.push('/model/' + currentModel + '/document/' + document._id)" :key="document._id">
+          <component
```

Comment:
Using `component` is unnecessary here. `<component :is>` is just for making it possible to switch the component being rendered based on `is`: https://masteringjs.io/tutorials/vue/dynamic-component. You should be able to use `<list-json>` here